SeasonOfDocs.html

```diff
@@ -61,10 +61,10 @@
    http://llvm.org/pubs/2002-12-LattnerMSThesis.html”Original
```

Comment:
This link is broken as well but I wasn't sure how to word it after updating the quotation marks 🤔